server/src/services/customers.ts

```diff
@@ -14,10 +14,26 @@
  * limitations under the License.
  */
 
-import {CustomerResponse} from '../interfaces';
+import {DEFAULT_CUSTOMER_PAYLOAD} from '../constants/default-payload';
+import {CustomerResponse, CustomerPayload} from '../interfaces';
 import {customerStorage} from '../storage';
 
 const getCustomer = async (customerId: number): Promise<CustomerResponse> =>
   await customerStorage.getCustomer(customerId);
 
-export default {getCustomer};
+/**
+ * Creates a customer if the customer is not already registered, and returns customer information.
+ * Returns existing customer information otherwise.
```

Comment:
customerStorage.addCustomer should throw error if customer already exists right

Reply:
@ifhuang Oh yea! This was the previous behaviour and I forgot to change the docstring. Will change it!